### ISSUE TITLE
(v9) PresenceBadge: add labels based on status prop

### ISSUE THREAD

**Issue by tringakrasniqi** [microsoft/fluentui#21137]
### Environment Information

- **Package version(s)**: v9 beta
- **Browser and OS versions**: all

### Describe the issue:

Currently the status prop of PresenceBadges is used only for its visual appearance, but this info should also be used to create labels for badges so they can be localized.
Based on this [RFC](https://github.com/microsoft/fluentui/pull/19258)

**Comment by smhigley** [comment]
Throwing some notes in here:

## Background:
The main motivation for including the strings in `PresenceBadge` is to have sensible accessibility defaults built into components. Since we have a `status` prop built in that controls the icon, IMO it should also control the alt text. It seems like a bit of an authoring gotcha to have `PresenceBadge` work perfectly for visual users with only the `status` prop, but then have screen reader accessibility silently fail unless authors to remember to include alternative text somewhere else, likely on the parent of the badge. They'd also need to separately control both the badge `status` and the label string.

## Proposal

We add a `strings` prop to PresenceBadge (probably also to Avatar) that uses the `PresenceBadgeStatus` type as keys. It should probably be optional, maybe with a `DEFAULT_STRINGS` english-language export from PresenceBadge.

IMO that sets us up well to have a consistent prop for internal strings across all components, and shouldn't get in the way of any future i18n solution we decide on.
